examples/templates/typographic-spacing.html

```diff
@@ -4,6 +4,253 @@
 category: _templates
 ---
 
+<!-- Headings preceding a block level element -->
+<div class="p-strip--light is-shallow">
+  <div class="row" style="max-width: 100%">
+    <p>Headings preceding block level element</p>
+  </div>
+  <div class="row" style="max-width: 100%">
+    <hr >
+  </div>
+  <div class="row" style="max-width: 100%">
+    <div class="col-2">
+      <h1>H1 gyp</h1>
```

Comment:
Why "gyp"?

Reply:
Just testing with descenders. I'll remove it.

Reply:
fixed